src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs

```diff
@@ -33,18 +42,24 @@ public sealed class OnlineGradientDescentTrainer : AveragedLinearTrainer<Regress
             + "In the TLC implementation of OGD, it is for linear regression.";
         internal const string ShortName = "ogd";
 
+        /// <summary>
+        /// Options for the <see cref="OnlineGradientDescentTrainer"/>.
+        /// </summary>
         public sealed class Options : AveragedLinearOptions
         {
             [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
             [TGUI(Label = "Loss Function")]
             internal ISupportRegressionLossFactory RegressionLossFunctionFactory = new SquaredLossFactory();
 
+            /// <summary>
+            /// A custom <a href="tmpurl_loss">loss</a>.
```

Comment:
>tmpurl_loss [](start = 34, length = 11)

should probably fix those before V1

Reply:
in April

---
In reply to: [268038547](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038547) [](ancestors = 268038547)